### ISSUE TITLE
Submissions of private resource repos fail on reload

### ISSUE THREAD

**Issue by DarrylBrooks97** [blaccsmith/website#39]
### Describe the bug

When a user submits a private repo url to the resource page, on reload shows a 500 error code

### To Reproduce

```shell
1. Go to /resources
2. Submit a url link of a private repo
3. Refresh page
```

### Expected Behavior

500 internal error

### Where is this happening

- [X] Desktop
- [X] Mobile

**Comment by DarrylBrooks97** [comment]
Resolved in 3289b4231044dd6ed05254eea1df8759fb010bcf

**Comment by blaccsmith** [comment]
Completed and closing
